apps/emqx_bridge_sqlserver/src/emqx_bridge_sqlserver_connector.erl

```diff
@@ -215,6 +215,7 @@ on_start(
         sql_templates => parse_sql_template(Config),
         resource_opts => ResourceOpts
     },
+    ok = emqx_resource:allocate_resource(ResourceId, pool_name, PoolName),
```

Comment:
the `PoolName` is same as `ResourceId`, so there should be no need to allocate it.
simply calling `emqx_resource_pool:stop(ResourceId)` is enough.